rasa_core/tracker_store.py

```diff
@@ -31,6 +31,38 @@ def __init__(self, domain, event_broker=None):
         self.domain = domain
         self.event_broker = event_broker
 
+    def check_attr(self, store):
+        if not hasattr(store, "port"):
+            store.port = None
+        if not hasattr(store, "db"):
+            store.db = None
+        if not hasattr(store, "username"):
+            store.port = None
+        if not hasattr(store, "password"):
+            store.password = None
+        if not hasattr(store, "timeout"):
+            store.timeout = None
+        return store
+
+    def find_tracker_store(self, store=None):
```

Comment:
I just realised that this should be class method. We are just using the `domain` attribute which could just be a parameter of `find_tracker_store`.